apps/admin/frontend/src/screens/tally/manual_tallies_form_screen.tsx

```diff
@@ -554,7 +593,10 @@ function ManualResultsDataEntryScreenForm({
                 variant="primary"
                 icon="Done"
                 onPress={saveResults}
-                disabled={setManualTallyMutation.isLoading}
+                disabled={
+                  someContestHasIncompleteResults ||
```

Comment:
No need to check as well for `someContestHasEmptyResults`?

Reply:
I think this code gets removed later, so it's not relevant, is that correct?

Reply:
Oop right, sorry about that. Commit by commit review issue on my end